src/app/chat/views/community_members_list.nim

```diff
@@ -114,17 +117,6 @@ QtObject:
       of CommunityMembersRoles.StatusType: result = newQVariant(self.memberStatus(communityMemberPubkey))
       of CommunityMembersRoles.Online: result = newQVariant(self.isOnline(communityMemberPubkey))
       of CommunityMembersRoles.SortKey: result = newQVariant(self.sortKey(communityMemberPubkey))
-      
-  proc rowData(self: CommunityMembersView, index: int, column: string): string {.slot.} =
```

Comment:
I think this is used to render suggestion (which is currently broken). So we should probably not delete that

Reply:
pls see: https://github.com/status-im/status-desktop/pull/3341

Reply:
I did not find any usage, but I trust your opinion and will bring it back. Thanks!

Reply:
Done.